test/models/search_facet_test.rb

```diff
@@ -0,0 +1,99 @@
+require "test_helper"
+
+class SearchFacetTest < ActiveSupport::TestCase
+
+  def setup
+    @search_facet = SearchFacet.first
+
+    # filter_results to return from mock call to BigQuery
+    @filter_results = [
+        {id: 'NCBITaxon_9606', name: 'Homo sapiens'},
+        {id: 'NCBITaxon_10090', name: 'Mus musculus'}
+    ]
+  end
+
+  # should return expected filters list
+  # mocks call to BigQuery to avoid unnecessary overhead
+  test 'should update filters list' do
+    puts "#{File.basename(__FILE__)}: #{self.method_name}"
+
+    # mock call to BQ, until a better library can be found
+    mock = Minitest::Mock.new
+    mock.expect :query, @filter_results, [String]
+
+    SearchFacet.stub :big_query_dataset, mock do
+      filters = @search_facet.get_unique_filter_values
+      assert_equal @filter_results, filters
+      mock.verify
```

Comment:
This is super-nit-picky, but I would reorder these, so that the checks mirror execution order.  First you mock.verify to confirm the method got called, then you assert_equal to confirm it returned what you expected

Reply:
Also makes sense.

Reply:
4d685ac